src/app/containers/PageHandlers/withData.test.jsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import { shouldShallowMatchSnapshot } from '../../../testHelpers';
-import { articleDataNews } from '../Article/fixtureData';
+import { articleDataNews, articleDataPersian } from '../Article/fixtureData';
```

Comment:
Now that there is custom articles logic in this file, can you please add tests to ensure that ws frontpages dont error :)

Reply:
@dr3 done!